app/src/main/java/ca/marshallasch/veil/DataStore.java

```diff
@@ -394,19 +390,13 @@ public void clearPeers() {
 
 
     public void runDataSaver(){
-        List<String> toBeDeletePostHashes = db.getAllPostsbyStatus(KnownPostsContract.POST_NORMAL);
-        List<String> toBeDeleteCommentHashes = db.getAllCommentsByStatus(KnownPostsContract.POST_NORMAL);
+        List<String> toBeDeletePostHashes = db.getAllPostsByStatus(KnownPostsContract.POST_NORMAL);
 
-        // delete all the post hashes from the hashtable
+        // delete all the post hashes from the hashtable along with its comments
```

Comment:
This  will not also delete the comments, just the posts. Access to the comments will be lost, but they will still be there.